tests/common/mod.rs

```diff
@@ -169,9 +179,13 @@ pub async fn init_sdk_get_init_result(
         users_device_private_key_bytes.try_into().unwrap(),
         users_signing_keys_bytes.try_into().unwrap(),
     );
+    let config = IronOxideConfig {
```

Comment:
Is there a reason this was specified and not just set to the default?

Reply:
Mostly because I don't know what the default will end up being. ;)